src/mapdata.h

```diff
@@ -447,6 +447,8 @@ struct submap {
     trap_id         trp[SEEX][SEEY];  // Trap on each square
     int             rad[SEEX][SEEY];  // Irradiation of each square
 
+    bool            known[SEEX][SEEY]; // has the player ever seen this tile
```

Comment:
I'm not sure what you mean. There's only 1 player and no way to switch to a different creature as far as I know.